src/visualizePath.js

```diff
@@ -0,0 +1,39 @@
+import { PIXELS_PER_TILE } from './constants';
+
+/* eslint no-param-reassign: ["error", { "ignorePropertyModificationsFor": ["renderer"] }] */
+function createPathSprites(path, renderer, cpt) {
```

Comment:
Good point. So one thing is that this function specifically modifies the `tagpro.renderer.pathSprites` object, so right now it is not generalizeable. For now, I think making it not generalizable results in cleaner, more readable code. If in the future we decide to draw more rectangles, we can refactor this to a helper and reuse it. For now, I will keep it.

I'll add the color and alpha as inputs though, since those are arbitrary and up to the user